test/index.test.js

```diff
@@ -175,6 +175,11 @@ const tests = [
     input: ".foo { animation-name: env(bar); }",
     expected: ":local(.foo) { animation-name: env(bar); }",
   },
+  {
+    name: "not localize animation-name in an global function",
+    input: ".foo { animation-name: global(bar); }",
+    expected: ":local(.foo) { animation-name: bar; }",
```

Comment:
Let's add a test case where `animation` used and animation name in different  places

Reply:
done 👍